### PR TITLE
Remove Gemnasium from README

### PR DESCRIPTION
**Pull request by nealedj**
Gemnasium has shut down https://docs.gitlab.com/ee/user/project/import/gemnasium.html

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,5 +1,5 @@
 # eQ Survey Runner
-[![Build Status](https://travis-ci.org/ONSdigital/eq-survey-runner.svg?branch=master)](https://travis-ci.org/ONSdigital/eq-survey-runner) [![codecov](https://codecov.io/gh/ONSdigital/eq-survey-runner/branch/master/graph/badge.svg)](https://codecov.io/gh/ONSdigital/eq-survey-runner) [![Dependency Status](https://gemnasium.com/badges/github.com/ONSdigital/eq-survey-runner.svg)](https://gemnasium.com/github.com/ONSdigital/eq-survey-runner) [![Codacy Badge](https://api.codacy.com/project/badge/Grade/82e63fc5bc5c43e8ba1ba6d13bfb4243)](https://www.codacy.com/app/ONSDigital/eq-survey-runner)
+[![Build Status](https://travis-ci.org/ONSdigital/eq-survey-runner.svg?branch=master)](https://travis-ci.org/ONSdigital/eq-survey-runner) [![codecov](https://codecov.io/gh/ONSdigital/eq-survey-runner/branch/master/graph/badge.svg)](https://codecov.io/gh/ONSdigital/eq-survey-runner) [![Codacy Badge](https://api.codacy.com/project/badge/Grade/82e63fc5bc5c43e8ba1ba6d13bfb4243)](https://www.codacy.com/app/ONSDigital/eq-survey-runner)
```
Comment (shahi645) [inline README.md:2]: @nealedj Why does `Codecov` change from a `Dependency status`, to a `Codacy Badge`?

Comment (nealedj) [reply README.md:2]: Does it? I'm seeing this:

screen shot 2018-06-13 at 13 04 31

Comment (shahi645) [reply README.md:2]: sorry was looking at wrong branch

**Review by shahi645 — APPROVED** [review]

**Review by jonnyshaw89 — APPROVED** [review]

**Comment by shahi645** [comment]
@nealedj FYI just rebased and updated with master

**Pull request merged**